src/Credentials/AssumeRoleWithWebIdentityCredentialProvider.php

```diff
@@ -99,7 +99,7 @@ public function __invoke()
             while ($result == null) {
                 try {
                     $token = is_readable($this->tokenFile)
-                        ? file_get_contents($this->tokenFile)
+                        ? @file_get_contents($this->tokenFile)
```

Comment:
:+1:  I agree that this calls should have a `@` in front, because it really doesn't need the error if it fails, and it does an explicit check itself afterwards.
Also see: https://github.com/aws/aws-sdk-php/pull/2245#issuecomment-875464278